contracts/feeds/PriceFeedIToken.sol

```diff
@@ -0,0 +1,26 @@
+pragma solidity 0.5.17;
+
+import "./IPriceFeedsExt.sol";
+import "../../interfaces/IToken.sol";
+import "../utils/SignedSafeMath.sol";
+
+
+contract PriceFeedIToken {
+    using SignedSafeMath for int256;
+
+    IPriceFeedsExt public priceFeedAddress; // underlying token Chainlink feed address
+    IToken public iTokenAddress;
+
+    constructor(IPriceFeedsExt _priceFeedAddress, IToken _iTokenAddress) public {
```

Comment:
public here gives a warning. no need to make constructor public there is no private constructors anyway

Reply:
actually I am wrong here